src/screens/BookDetails/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { View, Text, Image, ScrollView } from 'react-native';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+
+class BookDetails extends Component {
+  static navigationOptions = ({ navigation }) => {
+    const { params } = navigation.state;
+    return {
+      title: params ? params.bookTitle : 'Unknown'
+    };
+  };
+
+  render() {
+    const { navigation } = this.props;
+    const { author, imageLink, description } = navigation.getParam('book');
+    return (
+      <View style={styles.view}>
+        <ScrollView style={{ height: '100%' }}>
+          <View style={styles.imageView}>
+            <Image style={styles.image} source={{ uri: imageLink }} resizeMode="contain" />
+          </View>
+          <View style={styles.textView}>
+            <Text style={styles.author}>{`${author}`}</Text>
```

Comment:
In React, you can use any `JavaScript` between tags with \`\`.

In this case, you should be careful with template strings:
```js
<Text>{`${author}`}</Text>
```
Could be written like this:
```js
<Text>{author}</Text>
```

The difference in this case is the next one:

When you use template strings, you're always ending up with a `String`. This means that any value that you've received will be turned into a string, regardless of its type. If you received an `object`, `boolean`, `undefined`, `number`, etc. it will render into the string, and that shouldn't be happening.
In the other case, you can safely know React won't render any `falsy` values (except for values it can coerce to strings, like `0`, `NaN` and `Infinity`, because it escapes them to strings). That would also allow you to have some fallback in case of an error where the API sent a wrong value:

Could be written like this:
```js
<Text>{author || 'Fallback'}</Text>
```